### DISCUSSION TITLE
LogVulkanRHI: Error: Result failed, VkResult=-3

### DISCUSSION THREAD

**Discussion by zwhaa** [carla-simulator/carla#7517] [Q&A]
hello，when I run the command: make launch, the following error occurs：(ubuntu20.04，carla0.9.15，unreal engine4.26)

![1](https://github.com/carla-simulator/carla/assets/168153997/8f3b9c2c-2c2f-4431-854e-6d36c9938369)

[2024.04.29-03.11.58:977][  0]LogVulkanRHI: Error: Result failed, VkResult=-3
 at /home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/VulkanRHI/Private/VulkanSwapChain.cpp:530 
 with error VK_ERROR_INITIALIZATION_FAILED

[2024.04.29-03.11.59:364][  0]LogCore: Error: appError called: Fatal error: [File:/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/VulkanRHI/Private/VulkanUtil.cpp] [Line: 803] 
Result failed, VkResult=-3
 at /home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/VulkanRHI/Private/VulkanSwapChain.cpp:530 
 with error VK_ERROR_INITIALIZATION_FAILED
0x00007fa9097b952f libUE4Editor-VulkanRHI.so!VulkanRHI::VerifyVulkanResult(VkResult, char const*, char const*, unsigned int) [/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/VulkanRHI/Private/VulkanUtil.cpp:802]
0x00007fa9097c8708 libUE4Editor-VulkanRHI.so!FVulkanSwapChain::FVulkanSwapChain(VkInstance_T*, FVulkanDevice&, void*, EPixelFormat&, unsigned int, unsigned int, bool, unsigned int*, TArray<VkImage_T*, TSizedDefaultAllocator<32> >&, signed char, FVulkanSwapChainRecreateInfo*) [/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/VulkanRHI/Private/VulkanSwapChain.cpp:530]
0x00007fa9097e1c94 libUE4Editor-VulkanRHI.so!FVulkanViewport::CreateSwapchain(FVulkanSwapChainRecreateInfo*) [/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/VulkanRHI/Private/VulkanViewport.cpp:627]
0x00007fa9097e1990 libUE4Editor-VulkanRHI.so!FVulkanViewport::FVulkanViewport(FVulkanDynamicRHI*, FVulkanDevice*, void*, unsigned int, unsigned int, bool, EPixelFormat) [/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/VulkanRHI/Private/VulkanViewport.cpp:140]
0x00007fa9097e69f8 libUE4Editor-VulkanRHI.so!FVulkanDynamicRHI::RHICreateViewport(void*, unsigned int, unsigned int, bool, EPixelFormat) [/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/VulkanRHI/Private/VulkanViewport.cpp:991]
0x00007fa98687cac2 libUE4Editor-SlateRHIRenderer.so!FSlateRHIRenderer::CreateViewport(TSharedRef<SWindow, (ESPMode)0>) [/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/SlateRHIRenderer/Private/SlateRHIRenderer.cpp:359]
0x00007fa9e61f9ef0 libUE4Editor-SlateCore.so!SWindow::ShowWindow() [/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/SlateCore/Private/Widgets/SWindow.cpp:1397]
0x00007fa9d42d028b libUE4Editor-MainFrame.so!FMainFrameHandler::ShowMainFrameWindow(TSharedRef<SWindow, (ESPMode)0>, bool, bool) const [/home/zqht/UnrealEngine_4.26/Engine/Source/Editor/MainFrame/Private/Frame/MainFrameHandler.h:247]
0x00007fa9d42a4e62 libUE4Editor-MainFrame.so!FMainFrameModule::CreateDefaultMainFrame(bool, bool) [/home/zqht/UnrealEngine_4.26/Engine/Source/Editor/MainFrame/Private/MainFrameModule.cpp:292]
0x00007fa9e584d00a libUE4Editor-UnrealEd.so!EditorInit(IEngineLoop&) [/home/zqht/UnrealEngine_4.26/Engine/Source/Editor/UnrealEd/Private/UnrealEdGlobals.cpp:145]
0x000000000024a115 UE4Editor!GuardedMain(char16_t const*) [/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/Launch/Private/Launch.cpp:148]
0x00007fa9e38f7286 libUE4Editor-UnixCommonStartup.so!CommonUnixMain(int, char**, int (*)(char16_t const*), void (*)()) [/home/zqht/UnrealEngine_4.26/Engine/Source/Runtime/Unix/UnixCommonStartup/Private/UnixCommonStartup.cpp:264]
0x00007fa9e371b083 libc.so.6!__libc_start_main(+0xf2)
0x0000000000233229 UE4Editor!_start()

Signal 11 caught.
Malloc Size=65538 LargeMemoryPoolOffset=65554

**Comment by xavisolesoft** [comment]
Could you share what version of graphic card driver do you have installed, please?

**Reply by zwhaa** [reply]
nvidia 535.171.04

**Reply by zwhaa** [reply]
When I execute the command, this error occurs and the Unreal Engine cannot be opened, as shown in the following figure.

**Reply by zwhaa** [reply]
![Uploading 2024-05-06 08-51-06屏幕截图.png…]()
